CHANGELOG.md

```diff
@@ -1 +1,5 @@
 # Changelog
+## 3.0.0 / 2020-06-18
+
+* [Added] Add support for DCA and cluster checks for integrations autodiscovery. See [#119](https://github.com/DataDog/datadog-agent-boshrelease/pull/119).
```

Comment:
Could we add some explanation regarding the versioning change. Explain witch model we will use moving forward?

Reply:
Maybe, let's mention which version of the agent it embeds